ui/intermittent-failures/MainView.jsx

```diff
@@ -193,6 +193,9 @@ MainView.propTypes = {
   tableData: PropTypes.arrayOf(PropTypes.shape({})),
   graphData: PropTypes.arrayOf(PropTypes.shape({})),
   initialParamsSet: PropTypes.bool.isRequired,
+  user: PropTypes.shape({}).isRequired,
+  setUser: PropTypes.func.isRequired,
+  notify: PropTypes.func.isRequired,
```

Comment:
You shouldn't need to add props to files if they aren't being used in that file.